### DISCUSSION TITLE
Azure Files z uwierzytelnianie z AD i kompuer poza domena - da sie?

### DISCUSSION THREAD

**Discussion by MariuszFerdyn** [AzureCommunityPL/group#422] [Pomoc]
1. Ktomuś udalo zalogowac sie do Azure Files do komputera nie z domeny z wykorzystaniem poswiadczen AD lub AD DS?
2. A jezeli tak - to kto sie orjetuje, po jakich portach idzie komunikacja do AD? Bo jak komputer nie w domenie to przeciez nie kerberos ticket....

If your machine is not domain joined to an AD DS, you may still be able to leverage AD credentials for authentication if your machine has line of sight of the AD domain controller.

https://docs.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-active-directory-enable

**Comment by TomaszOledzki** [comment]
Nie sprawdzałem i tylko strzelam, więc trzeba to zweryfikować - obstawiam NTLM.

Edit: NTLM to byłby przy strzale do ADDS, dla AAD raczej wątpię. Dla uwierzytelnienia w oparciu o AAD musiałby to być oAuth.

Bardzo ciekawy case. Gdyby nikt nie napisał, a jeśli uda Ci się to dociec, daj proszę znać.

**Reply by MariuszFerdyn** [reply]
NTLM raczej... tylko, ze NTLM dla SQL lub IIS - nie wymaga kontaktu klienta z kontrolerem domeny.... a czemu tu wymaga... o ile wogole to dobre rozumienie tego.

**Reply by MariuszFerdyn** [reply]
A jednak nie.... https://serverfault.com/questions/455793/kerberos-authentication-for-workstations-not-on-domain - heh nie wiedzialem... Kerberos jest wydawany nawet dla komputerow nie bedacych w domenie.

**Reply by TomaszOledzki** [reply]
Super, dzięki za info.
